ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1415,6 +1415,39 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe
     workflow: rosa-aws-sts-hypershift
+- as: aws-rosa-sts-hypershift-guest-int-priv-link-full-f2
+  cron: 49 15 1,3,5,7,9,11,13,15,17,18,21,23,25,27,29 * *
+  steps:
+    cluster_profile: aws-sd-qe
+    env:
+      CHANNEL_GROUP: nightly
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @rosa and @network-ovnkubernetes and
+        @hypershift-hosted and not @fips'
+      OCM_LOGIN_ENV: integration
+      OPENSHIFT_VERSION: "4.14"
+      ROSA_LOGIN_ENV: integration
+      TAG_VERSION: '@4.14'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&;ROSA&
```

Comment:
private/private-link cluster is a disconnected cluster, I think we should use `~ConnectedOnly`.

Reply:
@kuiwang02 wdyt? e2e in rosa with tag `~ConnectedOnly`

Reply:
it depends on what the private-link cluster is.
if it can access from internal to outside, it is not disconnected. should use `~DisconnectedOnly&`
if it can not access from internal to outside, it is disconnected, should use `~ ConnectedOnly&`

note: normally private cluster is not disconnected. it means we can not access it from outside to internal directly and need proxy to access it. it does not mean it can not access from internal to outside.

Reply:
@heliubj18 @yasun1 
please double check if rosa-aws-sts-hypershift-private-link create disconnected cluster.

per definition of this workflow,
```yaml
    pre:
    - ref: aws-provision-vpc-shared
    - ref: aws-provision-tags-for-byo-vpc-ocm-pre
    - chain: aws-provision-bastionhost
    - ref: proxy-config-generate
    - chain: rosa-sts-oidc-config-create
    - chain: rosa-cluster-provision
    - ref: aws-provision-tags-for-byo-vpc
    - ref: osd-ccs-conf-idp-htpasswd-multi-users
    - ref: rosa-cluster-wait-ready-nodes
```
it seems it is private, but not disconnected.

not: private is not equal with disconnected.